src/scripts/helpers/swipe-tables.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+  'use strict';
+
+  var tableLeftPos, startSwipingPos, swipeRange;
+    
+  function handleSwipe (target) {
+    var mouseX = window.event.clientX;
+    var swipeDistance = startSwipingPos - mouseX;
+    if (swipeDistance <= swipeRange && swipeDistance >= (-1 * swipeRange)) {
+      target.scrollLeft += swipeDistance;
+    }
+  }
+  
+  function startSwiping (target) {
+    var offsets = target.getBoundingClientRect();
+    tableLeftPos = offsets.left;
+    startSwipingPos = window.event.clientX;
+    swipeRange = target.getElementsByTagName('table')[0].offsetWidth - target.offsetWidth;
+    target.onmousemove = function() {
```

Comment:
I think that using `target.onmouse...` makes it difficult for other parts of the code to attach events. Could [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) be used instead?

See main review comment for alternate suggestions that may not need this code